tests/reftests/list.test

```diff
@@ -230,6 +230,47 @@ rankers.1.0.0
 rankers.2.0.1
 rankers.2.0.7
 ### opam list --depends-on lacaml -s --all-versions --coinstallable-with phylogenetics.x
```

Comment:
coinstallability with a non-existent version used to return nothing. Now it returns what would be compatible with that version, which seems more useful